Src/CSharpier/DocPrinter/DocFitter.cs

```diff
@@ -116,8 +111,9 @@ void Push(Doc doc, PrintMode printMode, Indent indent)
                             : ifBreak.FlatContents;
 
                     Push(contents, currentMode, currentIndent);
+                    break;
                 }
-                else if (currentDoc is LineDoc line)
+                case LineDoc line:
                 {
```

Comment:
Do you need `{}` on all these cases?
Switch cases normally don't have them, so someone reading the code will think you're purposefully creating a scope because you're creating new variables that will conflict with the parent scope.

Reply:
I'm just used to adding the braces to switch branches if there are several statements in them. No matter whether there is variable declaration or not. 

I've removed them